client/state/rewind/backups/reducer.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Internal dependencies
+ */
+import { combineReducers, withoutPersistence } from 'state/utils';
+import {
+	REWIND_BACKUPS_RECEIVE,
+	REWIND_BACKUPS_REQUEST,
+	REWIND_BACKUPS_REQUEST_FAILURE,
+	REWIND_BACKUPS_REQUEST_SUCCESS,
+} from 'state/action-types';
+
+/**
+ * Returns the updated requests state after an action has been dispatched. The
+ * state maps site ID to whether a request is in progress.
+ *
+ * @param   {object} state  Current state
+ * @param   {object} action Action payload
+ * @returns {object}        Updated state
+ */
+export const requesting = withoutPersistence( ( state = {}, { type } ) => {
```

Comment:
Do we need to track requesting state? It should be okay to display slightly state data, even if we're in the process of requesting fresh information. The component should be able to infer this if items is equal to the default value of `{}`.

Reply:
Similarly managing multiple duplicate requests is already handled by the data layer. (Another reason why we used to store this).

Reply:
Remove the requesting state in f3275c3